src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -129,13 +129,13 @@
 
       var el = element[0];
       var ctrl = element.controller('mdFabSpeedDial');
-      var items = el.querySelectorAll('.md-fab-action-item');
+      var items = el.querySelectorAll(':scope > md-fab-actions > .md-fab-action-item');
```

Comment:
Not every browsers supports scoped stylesheets. See [the MDN reference](https://developer.mozilla.org/de/docs/Web/CSS/:scope#Browser_compatibility)

Especially all IE browsers aren't supported, but `ngMaterial` is supporting IE11